pkg/controller/performanceprofile/components/profile/profile.go

```diff
@@ -2,8 +2,8 @@ package profile
 
 import (
 	"fmt"
+	v1alpha1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1"
```

Comment:
same. Also, this import belongs in the group below.

Reply:
ok